src/test/java/net/sf/jabref/logic/layout/format/LatexToUnicodeFormatterTest.java

```diff
@@ -34,25 +34,27 @@
 
 public class LatexToUnicodeFormatterTest {
 
+    public final LatexToUnicodeFormatter formatter = new LatexToUnicodeFormatter();
+
     @Test
     public void testPlainFormat() {
-        assertEquals("aaa", new LatexToUnicodeFormatter().format("aaa"));
+        assertEquals("aaa", formatter.format("aaa"));
     }
 
     @Test
     public void testFormatUmlaut() {
-        assertEquals("ä", new LatexToUnicodeFormatter().format("{\\\"{a}}"));
-        assertEquals("Ä", new LatexToUnicodeFormatter().format("{\\\"{A}}"));
+        assertEquals("ä", formatter.format("{\\\"{a}}"));
+        assertEquals("Ä", formatter.format("{\\\"{A}}"));
     }
 
     @Test
     public void testFormatStripLatexCommands() {
-        assertEquals("-", new LatexToUnicodeFormatter().format("\\mbox{-}"));
+        assertEquals("-", formatter.format("\\mbox{-}"));
     }
 
     @Test
     public void testEquations() {
-        LayoutFormatter layout = new LatexToUnicodeFormatter();
+        LayoutFormatter layout = formatter;
```

Comment:
Fixed by changing the commit "Unify formatter tests" from 4f8fe59423896e1645 to 529bdab7aa900563